### PR TITLE
update readme and get started

### PR DESCRIPTION
**Pull request by chenkennt**
@vicancy

### CODE REVIEW

File: Documentation/tutorial/docfx_getting_started.md [hunk 1]
```diff
@@ -1,75 +1,74 @@
...
 
-1. Getting Started
+1. What is *DocFX*
 ---------------
 
-`docfx` is an API documentation generator for .NET, currently support C# and VB, as similar to JSDoc or Sphinx. It has the ability to extract triple slash comments out from your source code. What's more, it has syntax to link additional files to API to add additional remarks. `docfx` will scan your source code and your additional conceptual files and generate a complete HTML documentation website for you. `docfx` provides the flexibility for you to customize the website through templates. We currently have several embedded templates, including websites containing pure static html pages and also website managed by AngularJS. Of cause, if you are interested in creating your own website with your own styles, you can follow [how to create custom template](howto_create_custom_template.md) to create custom templates.
+*DocFX* is an API documentation generator for .NET, currently it supports C# and VB.
+It has the ability to generate API reference documentation from triple slash comments of your source code.
+What's more, it allows you to use markdown files to add additional content to the generated documentation.
```
Comment (vicancy) [inline Documentation/tutorial/docfx_getting_started.md]: also mention *conceptual documentation* here?

File: README.md [hunk 2]
```diff
@@ -8,11 +8,11 @@
...
 
 ## What is it?
-DocFX makes it extremely easy to generate your developer hub, complete with API reference, landing page, and how-to.
+*DocFX* makes it extremely easy to generate your developer hub with API reference, landing page, and how-to.
 There are currently two versions of the tool:
 
-* Windows specific IDE version which uses .NET Framework and works with [VS 2015 community](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
-* Cross platform console version which uses .NET Core and DNX
+1. Exe version which can be used as a command-line tool or inside VS IDE.
+2. DNX version which can be run croos platform.
```
Comment (vicancy) [inline README.md]: cross

File: Documentation/tutorial/docfx_getting_started.md (hunk 1 above)
Comment (chenkennt) [inline Documentation/tutorial/docfx_getting_started.md]: updated a little bit

**Comment by vicancy** [comment]
:+1:  LGTM

**Pull request merged**